rpc/tests/header.rs

```diff
@@ -36,8 +34,83 @@ async fn test_get_by_height() {
 }
 
 #[tokio::test]
-async fn test_get_by_height_non_existent() {
-    let client = test_client(AuthLevel::Read).unwrap();
+async fn get_by_height_non_existent() {
+    let client = new_test_client(AuthLevel::Read).await.unwrap();
 
     client.header_get_by_height(999_999_999).await.unwrap_err();
 }
+
+#[tokio::test]
+async fn get_by_hash() {
+    let client = new_test_client(AuthLevel::Read).await.unwrap();
+
+    let genesis_header = client.header_get_by_height(1).await.unwrap();
+    let genesis_header2 = client
+        .header_get_by_hash(genesis_header.hash())
+        .await
+        .unwrap();
+
+    assert_eq!(genesis_header, genesis_header2);
+}
+
+#[tokio::test]
+async fn get_verified_range_by_height() {
+    let client = new_test_client(AuthLevel::Read).await.unwrap();
+
+    let genesis_header = client.header_get_by_height(1).await.unwrap();
+    let second_header = client.header_get_by_height(2).await.unwrap();
+
+    let headers = client
+        .header_get_verified_range_by_height(&genesis_header, 3)
+        .await
+        .unwrap();
+
+    assert_eq!(headers.len(), 1);
```

Comment:
as per docs, this should return 2 headers, no?
https://node-rpc-docs.celestia.org/#header

Reply:
this is what i understood too, but it actually returns `(from, to)`. i think this is because you already have the `from` header.

Reply:
sure, that's what I thought. For me it's still the bug tho that's worth reporting

Reply:
it is an issue in the documentation: https://github.com/celestiaorg/go-header/issues/72